stellar-core-prometheus-exporter/stellar-core-prometheus-exporter.py

```diff
@@ -33,37 +36,125 @@ def duration_to_seconds(duration, duration_unit):
   return eval(time_units_to_seconds[duration_unit])
 
 class StellarCoreCollector(object):
-  def collect(self):
-    response = requests.get(args.uri)
-    json = response.json()
+  def __init__(self):
+    self.info_keys = ['ledger', 'peers', 'protocol_version', 'quorum', 'startedOn', 'state']
```

Comment:
I like that you are replacing the camel case metric names with underscored versions, should "startedOn" be replaced by "started_on" as well ?

Reply:
It is replaced but in the code below. I did not create mapping for those metrics because all but one don't need any changes.
I could for consistency add mapings here but I think it's not worth the hassle.

Reply:
perfect thanks, definitely not worth the hassle!